src/inference/include/ie/multi-device/multi_device_config.hpp

```diff
@@ -42,5 +42,11 @@ DECLARE_MULTI_CONFIG_KEY(DEVICE_PRIORITIES);
  * is not 0, the network will be dispatched to the strongest device.
  */
 DECLARE_CONFIG_KEY(AUTO_NETWORK_PRIORITY);
+
+/**
+ * @brief temp config option, to disable auto batching explicitly
+ *
+ */
+DECLARE_CONFIG_KEY(ALLOW_AUTO_BATCHING);
```

Comment:
do we need this?

Reply:
(after the global config key with the same name\meaning is merged to the master)

Reply:
@myshevts This PR was originally test in together with auto batching before your #9724 is merge to master:) So a lot of code related to batching itself is not necessary. the more accurate PR targeting for this specific loop issue is https://github.com/openvinotoolkit/openvino/pull/9765
I'm testing if any other performance drop side effect with this now, though theoretically I think should not in our use case(with not that big data size in queue)